fgpyo/sequence.py

```diff
@@ -70,3 +70,12 @@ def reverse_complement(bases: str) -> str:
         the reverse complement of the provided base string
     """
     return "".join([_COMPLEMENTS[b] for b in bases[::-1]])
+
+
+def gc_content(bases: str) -> float:
+    """Calculates the fraction of G and C bases in a sequence."""
+    if bases:
+        gc_count = sum(1 if (ch == "C" or ch == "G") else 0 for ch in bases)
+        return gc_count / len(bases)
+    else:
+        return 0
```

Comment:
1. Reminder to use `if len(str) == 0:` instead of the implicit conversion `if str:`.   This is included in my suggestion below.
2. Do we handle lower-case bases?  This is not included in my suggestion below.
```suggestion
    if len(bases) = 0:
        return 0
    gc_count = sum(1 if (ch == "C" or ch == "G") else 0 for ch in bases)
    return gc_count / len(bases)
```

Reply:
thank you! Suggestion implemented and I use .upper() on the input sequence, so now we should be handling lower and upper cases appropriately